CHANGELOG.md

```diff
@@ -80,42 +102,22 @@
 - Added support for `DataFrame.expanding` and `Series.expanding` for aggregations `count`, `sum`, `min`, `max`, `mean`, `std`, `var`, and `sem` with `axis=0`.
 - Added support for `DataFrame.rolling` and `Series.rolling` for aggregation `count` with `axis=0`.
 - Added support for `DataFrameGroupBy.get_group`.
-- Added support for `DataFrameGroupBy` and `SeriesGroupBy` aggregations `first` and `last`
 - Added support for `Series.str.match`.
 - Added support for `DataFrame.resample` and `Series.resample` for aggregations `size`, `first`, and `last`.
-- Added support for `DataFrameGroupBy.all`, `SeriesGroupBy.all`, `DataFrameGroupBy.any`, and `SeriesGroupBy.any`.
-- Added support for `DataFrame.nlargest`, `DataFrame.nsmallest`, `Series.nlargest` and `Series.nsmallest`.
-- Added support for `replace` and `frac > 1` in `DataFrame.sample` and `Series.sample`.
```

Comment:
based on our release cut change log, https://github.com/snowflakedb/snowpark-python/blob/release-v1.19.0/CHANGELOG.md most of those changes seems already in on the release branch

Reply:
Oh. I see. Our release notes are also not in sync here: https://github.com/snowflakedb/snowpark-python/releases/tag/v1.19.0

Reply:
Ok. I based the most recent changes off of the changelog in the branch as opposed to the release notes.